basics-assignment-3-start/src/app/display-details/display-details.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component } from '@angular/core';
+import { count } from 'rxjs';
+
+@Component({
+  selector: 'app-display-details',
+  templateUrl: './display-details.component.html',
+  styleUrls: ['./display-details.component.css']
+})
+
+export class DisplayDetailsComponent {
+  details: boolean = true;
+  counter: number = 0;
+  counterArray: number[] = [];
+  displayDetails() {
```

Comment:
No return type. 
Between the class and the new function there is supposed to be an empty line.
No need to redo just make sure to work correct next time :)